README.rst

```diff
@@ -119,6 +119,10 @@ The authentication micro-frontend also requires the following additional variabl
      - Enables the image layout feature within the authn. When set to True, this feature allows the inclusion of images in the base container layout. For more details on configuring this feature, please refer to the `Modifying base container <docs/how_tos/modifying_base_container.rst>`_.
      - ``true`` | ``''`` (empty strings are falsy)
 
+   * - ``TPA_UNLINKED_ACCOUNT_PROVISION_URL``
+     - URL to redirect to when the identity provided by third-party authentication is not yet linked to a platform account. This allows for redirecting to a custom sign-up flow handled by an external service to create the linked account. An empty string (the default) disables this feature.
```

Comment:
Apologies for holding this up, but it's still not clear to me how an account can get in this situation.  It would be a big help if you could describe the logistration flow that benefits from this (maybe with a diagram?), and why one of the existing flows is not sufficient.

Reply:
On a more general note, shouldn't this be a per-provider thing, as opposed to a single URL for everything?

Reply:
@arbrandes In this particular use case, we don't want registration to be handled by Open edX from third-party auth (it will be handled by an external service which then creates the appropriate user account in the Open edX platform via API calls), so if the user tries to log in we end up in the situation where we have third-party auth and don't know which user account it should link to (because there's not one yet, really). Instead of the default behaviour of prompting the user to manually link an existing account, this would then redirect to the aforementioned external service.

While the concrete use case only has one provider, I think it still makes sense to have a single URL if there were more.

If this explanation is sensible enough, I can update the readme.